Find-and-Replace/Replace-text-in-headers-and-footers/.NET-Framework/Replace-text-in-headers-and-footers/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.Drawing;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Replace_text_in_headers_and_footers
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+                //Loads the template document
```

Comment:
change the XML comments as per feedbacks

Reply:
This has been addressed